modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc

```diff
@@ -34,18 +40,21 @@ To overwrite it, you can add a custom port numbers greater than 1024 to [path]``
 ssh_push_port_https = high_port
 ----
 
-After changing this configuration parameter you must restart services
-such as [systemitem]``salt-secrets-config.service``,
-[systemitem]``tomcat.service``, and
-[systemitem]``taskomatic.service``.  To cover all needed services, it
-is the best to restart [command]``spacewalk-service`` as root:
+After changing this configuration parameter you must restart services such as [systemitem]``salt-secrets-config.service``, [systemitem]``tomcat.service``, and
+[systemitem]``taskomatic.service``.
+To cover all needed services, it is the best to restart the {productname} Server as root:
 
 ----
-spacewalk-service restart
+mgradm restart
```

Comment:
Above you said run inside of the container. Inside of the container `spacewalk-service restart` is correct.
`mgradm restart` will restart the whole container and need to be run outside.

Maybe you describe both like:

> Inside the container run `spacewalk-service restart` or as alternative you can also restart the whole container when we run on the host `mgradm restart`.

... or similar.

Reply:
yes, I changed it for clarity.